xcconfigs/Project.xcconfig

```diff
@@ -21,6 +21,7 @@ CBL_VERSION_STRING                 = 3.1.0
 CBL_BUILD_NUMBER                   = 0
 CBL_EXPORTED_SYMBOLS_FILE          = Objective-C/CouchbaseLite.exp
 CBL_SWIFT_MODULEMAP_FILE           = Swift/CouchbaseLiteSwift.modulemap
+CBL_COPYRIGHT_YEAR                 = 2020
```

Comment:
Should it be 2021?

Reply:
I kept that for testing, forget to revert it.
```suggestion
CBL_COPYRIGHT_YEAR                 = 2021
```